desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -108,21 +108,19 @@ const PathBrowser: React.FC<PathBrowserProps> = ({
       <>
         {!isEditMode ? (
           <div className="hue-path-browser" data-testid={`${testId}`}>
-            {showIcon ? (
+            {showIcon && (
               <div
                 className="hue-path-browser__file-system-icon"
                 data-testid={`${testId}__file-system-icon`}
               >
                 {icons[extractFileSystem(breadcrumbs[0].label)]}
               </div>
-            ) : (
-              <></>
             )}
             <div className="hue-path-browser__breadcrumbs" data-testid={`${testId}-breadcrumb`}>
               {breadcrumbs.length <= 3 ? (
```

Comment:
nit: same here, you can use:` breadcrumbs.length <= 3 && (`

Reply:
I believe that the ternary operator is more apt here as breadcrumbs.length <= 3 will render only the breadcrumbs else breadcrumbs with dropdown is rendered